docs/source/metrics.rst

```diff
@@ -256,12 +256,15 @@ Complete list of metrics
     - :class:`~ignite.metrics.MetricsLambda`
     - :class:`~ignite.metrics.Precision`
     - :class:`~ignite.metrics.Recall`
+    - :meth:`~ignite.metrics.metric.reinit__is_reduced`
```

Comment:
actually, these methods are already added to the docs in the end, see here lines 326 and 328.

Reply:
@vfdev-5 I already updated the order in the metric list :)

Reply:
@josselineperdomo ops, sorry, I misunderstood what was done. However, `reinit__is_reduced` and `sync_all_reduce` are not metrics but just helpers. So, let's remove them from the list and keep the order as it was:
```
.. autofunction:: sync_all_reduce

.. autofunction:: reinit__is_reduced
```

Reply:
Sorry! I misunderstood and I thought I had to put in the metric list the decorators. My last commit fixed it